CMakeLists.txt

```diff
@@ -0,0 +1,85 @@
+cmake_minimum_required(VERSION 2.8)
+
+project(ITK_BENCHMARK)
```

Comment:
Call the module

  ITKPerformanceBenchmarks

Call the benchmarks CMake project (living in the "examples" directory)

  ITKBenchmarks